_infra/docker/Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM openjdk:17-slim
+FROM eclipse-temurin:17-jre-alpine
 
 ARG JAR_FILE=case.jar
-RUN apt-get update
```

Comment:
`apt-get` isn't installed in alpine, but we don't install anything in this image anyway.